### PR TITLE
Add RTK v2 support

### PR DESCRIPTION
**Pull request by dacuster**

### CODE REVIEW

File: src/microstrain_config.cpp [hunk 1]
```diff
@@ -608,6 +608,9 @@ bool MicrostrainConfig::configureRTK(RosNodeType* node)
       }
     }
 
+    // set the GNSS channel fields
+    inertial_device_->setActiveChannelFields(mscl::MipTypes::DataClass::CLASS_GNSS3, supportedChannels);
```
Comment (robbiefish) [inline src/microstrain_config.cpp:612]: I would pull the `main` branch into your fork as this has already been updated and should not be showing up as a difference

**Review by robbiefish — CHANGES_REQUESTED** [review]
Looks mostly good for ROS, but does this does not appear to have changes required for ROS2 yet. This just requires adding the proper `using` statements into the `microstrain_defs.h` file

**Comment by dacuster** [comment]
Forgot this was common to ROS2 as well. I'll work on that shortly

**Review by robbiefish — CHANGES_REQUESTED** [review]
This now works well for both ROS and ROS2.

I would just add some documentation to the [`rtk_dongle_enable` parameter in params.yml](https://github.com/LORD-MicroStrain/microstrain_inertial_driver_common/blob/main/config/params.yml#L123) that explains that enabling this option will also cause the node to publish on one of two different topics depending on the version of the RTK dongle connected to the GQ7

**Review by robbiefish — APPROVED** [review]

**Pull request merged**